nixos/modules/services/web-apps/mealie.nix

```diff
@@ -50,7 +50,7 @@ in
   };
 
   config = lib.mkIf cfg.enable {
-    systemd.services.mealie = {
+    systemd.services.mealie = rec {
```

Comment:
The `rec` is required in order to allow referencing the `environment` key from within the same set.

Reply:
Using `config` as a module argument and referencing `config.path.to.the.option` should be preferred -- it deals better with the module system's merging behaviour.